cloudinit/cmd/main.py

```diff
@@ -786,7 +786,9 @@ def status_wrapper(name, args, data_d=None, link_d=None):
         else:
             errors = ret
 
-        v1[mode]["errors"] = [str(e) for e in errors]
+        v1[mode]["errors"] = []
+        for e in errors:
+            v1[mode]["errors"].append(str(e))
```

Comment:
What is the purpose of this change? The new code is functionally identical to the code it replaces.

Reply:
`errors` can be an integer value here, not always a list.

Reply:
Indeed there was in issue in the code change, fixed it. Thanks.